src/nnn.c

```diff
@@ -190,6 +190,7 @@
 #define ASCII_MAX       128
 #define EXEC_ARGS_MAX   10
 #define LIST_FILES_MAX  (1 << 14) /* Support listing 16K files */
+#define LIST_INPUT_MAX  (1 << 28) /* Support reading in 256MiB */
```

Comment:
Any specific reason for picking `256MiB` or just arbitrarily chosen limit?

Reply:
The original limit was 65536 paths each with `PATH_MAX == 4096` bytes, so 256 MiB in total. It seemed to me that @jarun wanted to keep it in case of invalid paths or something like that even though that's a bit overkill. I mean all of these numbers are arbitrary. Why did I originally pick 65k files? I don't know. It seemed like a nice number.

Reply:
The manpage says the limit is 64MiB

https://github.com/jarun/nnn/blob/master/nnn.1#L315

So looks like this was just a mistake. I think we should change this to:

```suggestion
#define LIST_INPUT_MAX  (LIST_FILES_MAX * PATH_MAX)
```

Just like the manpage says.

Reply:
He might have changed it afterwards. That's fine.